### ISSUE TITLE
Aufruf über FHEM funktioniert nicht, da Dateien im falschen Verzeichniss liegen

### ISSUE THREAD

**Issue by Jambo52** [tillsteinbach/WeConnect-mqtt#294]
Hallo Till
Nachdem ich weconnect mit FHEM auf meinem "Test Rpi" ans laufen bekommen habe, habe ich Stück für Stück begonnen das Projekt auf meinem "Haupt RPI" , mit dem ich die ganze Haussteuerung mache, zu installieren. Der normale Aufruf über die Konsole funktioniert, aber der Aufruf über FHEM geht nicht, auch nicht mit exater Pfadangabe.
`"sudo /usr/local/bin/weconnect-mqtt --mqttbroker 192.168.xxx.xx --netrc /home/pi/.netrc --no-capabilities --convert-times Europe/Berlin --update-on-connect"`
Fhem meldet im Log, das einige Datein nicht gefunden werden. Die gesuchten Dateien befinden sich im Ordner : /home/pi/.local/lib/python3.9/side-packages. Die Dateien werden aber beim Aufruf über FHEM im Verzeichniss /usr/local/lib/python3.9/dist-packages gesucht. Dort sind aber die benötigten Python Dateien nicht vorhanden.
Selbst bei einem Upgrade von weconnect landen die neuen weconnect-Datein im side-package Ordner. Ich habe daraufhin alle Dateien die das Installationsdatum hatten von dort aus in den dist-packages Ordner kopiert und der Aufruf über FHEM funktioniert dann einwandfrei.
Ich vermute es liegt daran, das ich vor der Installation von weconnect ein update/upgrade des RPI vorgenommen habe, welches gefühlt länger als die Neuinstallation gedauert hat, während der "Test Rpi" noch im Erst- Installationszustand von bullseye ist.

Vielleicht kannst du das mal nachvollziehen, denn auch beim upgrade landen die neuen Dateien in einem anderen Ordner wie die ursprünglich installierten, die dann dort wie bisher weitergenutzt werden.

Gruß
Nobbi

**Comment by tillsteinbach** [comment]
Kann es sein dass mehrere python Versionen installiert sind?

**Comment by Jambo52** [comment]
Nein, ich habe auf beiden Raspis bullseye Anfang des Jahres installiert. Auf beiden wird auch nur die Version 3.9.2 angezeigt.
Nur bei einem Raspi habe ich dieses Update und Upgrade gestartet und der hat die falschen Verzeichnisse. In dem nicht upgrade Raspi sind im side-packages Ordner keine Python Dateien vorhanden, sondern alle im dist-package Ordner, auch die weconnect-Dateien.
